packer/build.pkr.hcl

```diff
@@ -7,7 +7,9 @@ build {
     host_alias = "${var.ansible_host_alias}"
     playbook_file = "${var.playbook_file_path}"
     extra_arguments  = [
-      "-e", "aws_region=${var.aws_region}"
+      "-e", "aws_region=${var.aws_region}",
+      "-e", "s3_bucket=${var.aws_s3_release_bucket}",
+      "-e", "nagios_api_key=${var.nagios_api_key}"
```

Comment:
Can remove Nagios key variable.

Reply:
As discussed, Nagios config will be retained for now and will be reviewed in the future as necessary.